conf/config.toml

```diff
@@ -67,7 +67,7 @@ leader-schedule-limit = 4
 region-schedule-limit = 1024
 replica-schedule-limit = 1024
 merge-schedule-limit = 8
-tolerant-size-ratio = 0
+# tolerant-size-ratio = 0.0
```

Comment:
Why do we need to update this line?

Reply:
```
[FATAL] [main.go:54] ["parse cmd flags error"] [error="toml: cannot load TOML value of type int64 into a Go float"
```
The original configuration file cannot be parse.